src/localStorage.js

```diff
@@ -1,8 +1,21 @@
-const tasks = JSON.parse(localStorage.getItem('task.list')) || [];
-const taskKey = 'task.list';
+const taskKey = 'tasks';
+const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
 
-function save() {
+function save(tasks) {
   localStorage.setItem(taskKey, JSON.stringify(tasks));
 }
 
-export { tasks, taskKey, save };
\ No newline at end of file
+function removeFromStorage(id) {
+  tasks.forEach((task, index) => {
+    if (task.id === id) {
+      tasks.split(index, 1);
```

Comment:
- [ ] Here as well. I would recommend using `es6` syntax as it is a project requirement that helps you in writing more clean code using ES6 syntax.